### ISSUE TITLE
Directories not being synced

### ISSUE THREAD

**Issue by labkode** [owncloud/client#6783]
Hi,

We have seen some directories not being synced.
This is the output of the sqlite database on the client.
```
sqlite> select * from metadata where path like 'ttt%';
365117602342423491|8|ttt/docs|0|0|0|0|1517483710|2|1910ed:1533871806.629|1642733|WDNVCKR|0|1||0
7214500622853205675|8|ttt/site|0|0|0|0|1517483710|2|1910ee:1533871806.751|1642734|WDNVCKR|0|1||0
-1201754025941583126|7|ttt/fff|39321773|0|0|0|1537776833|0|25359908740268032:008e004c|25359908740268032|WDNVCKR|2|0|7d157d7c000ae27db146575c08ce30df893d3a64|1
-7098688356274325768|3|ttt|39321770|0|0|0|1537776834|2|18de8b:1537776834.583|1629835|WDNVCKR|0|1||0
```

The original situation was the following:

On the client:
No ttt directory synced on the client.

On the server:
ttt
ttt/docs
ttt/site

Then we put on the server the fff file and the directory ttt was created and the fff file was downloaded.

The other directories are not yet synchronized. 

Client version:  2.4.1 on Ubuntu
Server version: our own version (CERNBox)

@ogoffart @moscicki

**Comment by guruz** [comment]
> Client version: 2.4.1 on Ubuntu

Please try 2.5.0

**Comment by ogoffart** [comment]
I do not quite understand the bug.
Do you have steps to reproduce?

**Comment by guruz** [comment]
Can this be related to recent issues discovered/fixed by @jnweiger @ckamm @ogoffart ? But those were related to new discovery afaik.

**Comment by guruz** [comment]
@labkode ping..

**Comment by labkode** [comment]
@ogoffart We cannot reproduce the state on the db, what we can reproduce is that the run of the sync client does not synchronise the changes. 

@guruz We can re-test with 2.4.2, 2.5 is not yet branded, I let you know,

Thanks

**Comment by ogoffart** [comment]
Anything in the log tells why they are not synchronized?  Are there any error, are they ignored?
A log file would be usefull.

**Comment by guruz** [comment]
@labkode Any log or answer is appreciated, maybe we can get a fix still into 2.5.1. thanks

**Comment by ogoffart** [comment]
Closing this issue as this cannot be reproduced.
